app/units/units.css

```diff
@@ -111,6 +118,13 @@ span.icon {
     color: #DD432B;
     font-size: 35px;
     z-index: 2;
+    cursor: pointer; /* Add cursor pointer */
+    user-select: none; /* Make it unselectable */
+}
+
+span.icon:hover {
+    color: black; /* Change color on hover */
+    background-color: rgba(0, 0, 0, 0.1); /* Add hover effect highlight */
```

Comment:
Could you remove the background colour as it was not part of the issue? Thanks :)

Reply:
Sure I will change it as per your requirement @jennifermtan

Reply:
<img width="982" alt="Screenshot 2024-06-02 at 10 22 01 AM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/58948980/ed19c32f-6628-43ee-92f2-f5db75877c7d">
is this okay for you? @jennifermtan

Reply:
Looks great! Maybe decrease the size of the circle (similar to the image I attached previously) and decrease opacity of the colour to increase contrast. Let me know if you have any other ideas as well :)

Reply:
The size of the circle is like the image that you have attached but here as per the theme it is highlighting that the reason it is looking larger.
Not only circle the combination of the circle and the icon should be smaller as like the Screenshot.

<img width="990" alt="Screenshot 2024-06-02 at 11 04 39 AM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/58948980/ae12f02f-3fea-4444-bb57-c14eb04a9f29">

is this okay with you ? @jennifermtan

Reply:
Nice catch! Let's keep the icon the same size as before but love the highlight :D Thanks so much!

Reply:
I have done a new PR, review the request

Reply:
Please do not make a new PR and commit your changes to this branch.

Reply:
Committed the changes to current brunch

Reply:
Sorry I might've not been too clear but we only want the circular background when the icon is hovered on. It shouldn't be visible when it is not hovered on. Other than that code looks good! I'll approve after you make this last change :) @sapta19

Reply:
Looks very good @sapta19. Just this above change.

Reply:
I also suggest that the border-radius should match the other UI buttons. However, I will leave that to @jennifermtan's instruction.

Reply:
I have changed that @jennifermtan has suggested

Normal State
<img width="985" alt="Screenshot 2024-06-02 at 9 23 14 PM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/58948980/ea83a57f-ab88-4d6d-adcc-fb1ed6dcbb4c">
Hovered State
<img width="1007" alt="Screenshot 2024-06-02 at 9 24 15 PM" src="https://github.com/GoodGameRuler/unitrack_v3/assets/58948980/336e5752-8591-4bbc-9455-7d11ce949856">